prerequisites/software.rst

```diff
@@ -1,6 +1,60 @@
 Software
 ********
 
+1. Client requirements
+======================
+
+Please note that, while Zammad being a web application, there's some
+requirements for your clients. This ensures that Zammad works as expected.
+
+1.1. Supported Browsers
+-----------------------
+
+.. list-table:: Zammad/Browser version compatibility
+   :header-rows: 1
+   :widths: 20, 30
+
+   * - Firefox 78+
+     - (ESR)
+   * - (Google) Chrome 83+
+     - This also applies for all Chromium based Browsers like Microsoft Edge
+   * - Opera 69+
+     - (based on Chromium 83)
+   * - Microsoft Internet Explorer 11
+     - 
+   * - Safari 11
+     - 
+
+.. note::
+
+   | Zammad heavily uses Javascript which makes it a hard requirement.
+   |
+   | Addons that hook into page content may interfere with Zammads function
```

Comment:
Will users understand about what you mean by Addons? Since there are no public addons available, we might scratch this sentence.
IMO it should be obvious that if people change stuff in the application, this can cause bugs.

Reply:
Oh sorry! I was referring to browser addons, not Zammad addons!
We regulary get support request because of e.g. google translator translating "open" to "beleidigen" - who knows what Google thinks at that point haha.

Reply:
→ Will add Browser Addon